packages/@yoda/endoscope/exporter/flora.js

```diff
@@ -0,0 +1,15 @@
+var Agent = require('@yoda/flora').Agent
+
+class Exporter {
+  constructor (name) {
+    this.name = name
+    this.agent = new Agent('unix:/var/run/flora.sock')
```

Comment:
Could we have a method like `Agent.getInstance()`? So that we don't specify the uri everywhere if we want to change.

Reply:
Moved to https://github.com/yodaos-project/node-flora/issues/11